core/unit_test/TestGraph.hpp

```diff
@@ -21,9 +21,17 @@
 
 namespace Test {
 
-struct NoOpFunctor {
-  template <class... Ts>
-  KOKKOS_FUNCTION void operator()(Ts&&...) const noexcept {
+template <class ExecSpace, class ValueType>
+struct NoOpReduceFunctor {
+  KOKKOS_FUNCTION void operator()(int, ValueType&) const {
+    Kokkos::abort("Should never be called!");
+  }
+  KOKKOS_FUNCTION void operator()(int, int, ValueType&) const {
+    Kokkos::abort("Should never be called!");
+  }
+  KOKKOS_FUNCTION void operator()(
+      const typename Kokkos::TeamPolicy<ExecSpace>::member_type&,
+      ValueType&) const {
```

Comment:
What was wrong here?

Reply:
The Cuda backend implementation for TeamPolicy reductions required value type deduction from the functor and that doesn't work if the functor takes arbitrary arguments. That's something we might be able to fix down the road but it doesn't feel worth it for Graph tests.

Reply:
OK